### PR TITLE
packages/vsphere/docs: Improve logs collection documentation

### PR DESCRIPTION
**Pull request by alstolten**
## Proposed commit message

More details on vSphere integration documentation

Clarified that vSphere integration log collection works through a syslog daemon created by the agent running the integration and explained how to connect vSphere to it.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Comment by alstolten** [comment]
:+1:

**Comment by ishleenk17** [comment]
Thanks for the update. 
There is a CI failure. 
Please do the documentation change in [this file](https://github.com/elastic/integrations/blob/main/packages/vsphere/_dev/build/docs/README.md) and then run elastic-package build.
The README under docs gets autogenerated

File: packages/vsphere/docs/README.md [hunk 1]
```diff
@@ -441,7 +441,8 @@ An example event for `datastore` looks as following:
 
 ## Logs
 
-To access the logs, host address (localhost) and host port (9525) needs to be passed in Kibana UI. 
+Log collection works through a syslog daemon. You need to configure the listening host address/IP (default: localhost) and port (default: 9525) in the integration. After that you need to configure vSphere to send logs to a remote syslog host and provide the configured hostname/IP and port of the Elastic Agent host.
+
```
Comment (shmsr) [inline packages/vsphere/docs/README.md:445]:
```suggestion
```

Comment (shmsr) [inline packages/vsphere/docs/README.md:445]:
```suggestion
To collect logs, a syslog daemon is used. First, you must configure the listening host/IP address (default: localhost) and host port (default: 9525) in the integration. Then, configure vSphere to send logs to a remote syslog host and provide the configured hostname/IP and port of the Elastic Agent host.
```

**Comment by shmsr** [comment]
Please update CHANGELOG as well.

File: packages/vsphere/docs/README.md (hunk 1 above)
Comment (alstolten) [reply packages/vsphere/docs/README.md]: Thanks, done!

**Comment by alstolten** [comment]
Hi @shmsr I think I got it now, bear with me, it is my first contribution to integrations I believe. There was a very similar parallel effort on https://github.com/elastic/integrations/pull/8859 which I was not aware of. However I believe mentioning the syslog daemon is still beneficial. WDYT @alaudazzi ?

File: packages/vsphere/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.11.1"
+  changes:
+    - description: Add more specific docs to vSphere logs configuration.
```
Comment (shmsr) [inline packages/vsphere/changelog.yml:4]:
```suggestion
    - description: Add more specific details to vSphere logs configuration documentation.
```

**Comment by shmsr** [comment]
@alstolten Changes were made only in `packages/vsphere/_dev/build/docs/README.md` but the same changes must reflect in `packages/vsphere/docs/README.md` too and that's why CI was failing. I have fixed the issue.

How did I do it? I ran `elastic-package build` to achieve the same. This command runs lints, formats the code, makes necessary changes, and also builds the package.

**Review by shmsr — APPROVED** [review]

**Pull request merged**